recipe/meta.yaml

```
@@ -1,52 +1,54 @@
-{% set version = "2.1.0" %}
+{% set version = "2.3.0" %}
 
 package:
   name: pyviz_comms
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/p/pyviz_comms/pyviz_comms-{{ version }}.tar.gz
-  sha256: f4a7126f318fb6b964fef3f92fa55bc46b9218f62a8464a8b18e968b3087dbc0
+  sha256: 44709b2e639e84eb1ae7be5eaf3e28a9be7932d29a8cefce1cd603cb3904ad9c
 
 build:
   number: 0
   noarch: python
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script:
+    - {{ PYTHON }} -m pip install . --no-deps -vv --no-build-isolation
 
 requirements:
   host:
-    - python >=2.7
+    - python
     - param
     - pip
-    - jupyter-packaging >=0.7.9
-    - jupyterlab >=3.0
-    - notebook
-    - setuptools
-    - twine
-    - rfc3986
-    - keyring
+    - jupyter-packaging >=0.7.9,<0.8
+    - jupyterlab >=3.0,<4
+    - setuptools >=40.8.0
     - wheel
   run:
-    - python >=2.7
+    - python >=3.7
```

```suggestion
    - python
```
As we build as an arch-specific python package then it should be a plain `python` (without pinning) in `run`. Above we skip all pythons `<3.7` by the `build/skip` key